docs/map-making/upgrading-1-8-maps.md

```diff
@@ -1,88 +1,147 @@
 A list of (case sensitive) changes, to upgrade a map from a TripleA version to another one.
 
 ## Upgrade from 1.8.0.9 to 1.9.0.0.3266
-(old:new)
+
 Note: The following list covers all basic upgrade cases for maps, but it may not upgrade "Property" options (playerProperty, unitProperty...) of already deprecated items.
+
 Note: Before applying the changes below (assuming you do it with a simple replace), you should make sure to not have unnecessary spaces inside angle brackets (chevrons).
-Note: Some items are not validated; thus, if your game plays on 1.9.0.0.3266, this doesn't necessarily mean that you correctly upgraded it.
 
-### Set:
+Note: Some items are not validated; thus, if your game plays on 1.9.0.0.3266, this doesn't necessarily mean that you correctly upgraded it (for more information, see the "warning" section).
+
+### New XML settings:
+
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
+
+`<triplea minimumVersion="*"/>`:`<triplea minimumVersion="1.9.0.0"/>`
+
+(The "*" means whatever)
+
+### Not already deprecated XML changes:
 
-`<triplea minimumVersion="1.9.0.0"/>`
+_(these are the main changes on game file items that were not deprecated, so you normally need to change these only, within the game file)_
 
-### Not already deprecated xml changes:
-_(you normally need to change these only)_
+Within every game file (XML), inside the "games" folder inside the main folder of the map, change all occurrences of the code before the colon with the code after the colon.
 
 `attatchment`:`attachment`
```

Comment:
pre-pending items with a dash or asterisk will create a list, I think that might be the formatting looked for. Adding in the extra line breaks works too, though not as compact.